src/ui/public/agg_table/agg_table.js

```diff
@@ -91,6 +92,12 @@ define(function (require) {
               formattedColumn.class = 'visualize-table-right';
             }
 
+            if (field.type === 'number') {
+              formattedColumn.total = table.rows.reduce(function (prev, curr, n, all) {
```

Comment:
You could use lodash's `_.sum` method here, maybe `_.sum(table.rows, [i, 'value'])`